### PR TITLE
Add IV hexagon graph (attempt 2)

### PR DESCRIPTION
**Pull request by RedstonewolfX**
Displays a Pokémon's IVs by modifying the IV hexagon.
Removes the IV label as it may be covered up by the hexagon

Should work, but needs to be tested as I have no way to do that right now

### CODE REVIEW

File: src/components/PokemonCard.tsx [hunk 1]
```diff
@@ -4,6 +4,10 @@ import caughtImage from "../images/caught.png";
 // @ts-ignore
 import uncaughtImage from "../images/uncaught.png";
 
+export function createPoly(health: number, attack: number, defense: number, sp_atk: number, sp_def: number, speed: number): string {
```
Comment (dewinterjack) [inline src/components/PokemonCard.tsx]: no need to export this

**Comment by dewinterjack** [comment]
Looks ok to me but will need to run it to see if it works before merging

**Comment by RedstonewolfX** [comment]
Yep, testing is what I was hoping for

File: src/components/PokemonCard.tsx (hunk 1 above)
Comment (RedstonewolfX) [reply src/components/PokemonCard.tsx]: Removed the export statement

**Review by dewinterjack — APPROVED** [review]
Tested it and it's working! 🔥

**Pull request merged**